cli/connect/connect.go

```diff
@@ -97,23 +97,23 @@ func Eval(connectCtx ConnectCtx, connOpts connector.ConnectOpts, args []string)
 	}
 	defer conn.Close()
 
-	var eval string
-	evalArgs := []interface{}{command}
+	evalArgs := []interface{}{command, connectCtx.Language == SQLLanguage}
 	if connectCtx.Language != DefaultLanguage {
 		// Change a language.
 		if err := ChangeLanguage(conn, connectCtx.Language); err != nil {
 			return nil, fmt.Errorf("unable to change a language: %s", err)
 		}
```

Comment:
The change will help to be clear:
```suggestion
		}
		evalArgs = append(evalArgs, false)
```

Reply:
Updated.